.travis.yml

```diff
@@ -75,7 +75,7 @@ script:
   - travis_wait ../project/script/validate/vendor
   - GOOS=linux script/setup/install-dev-tools
   - go build -i .
-  - make check
```

Comment:
It looks like this needs to be skipped for Darwin

Reply:
Changed to do lower on, and with a check for `TRAVIS_GOOS=linux`